### DISCUSSION TITLE
各位手搓配置的大佬们如何监测服务端各节点的流量？

### DISCUSSION THREAD

**Discussion by Benjamin1919** [XTLS/Xray-core#3781] [Q&A]
服务端不用什么UI面板，只用core手搓配置，有多个入站节点，应该如何直观地监测各节点的流量使用情况呢？

服务端装了哪吒探针，可惜只能查看VPS总体的入站和出站流量。
目前暂时通过xray的metrics查看各节点的使用量 ( https://xtls.github.io/config/metrics.html#expvars )，但是vars导出的数据可读性太差，而且不方便实时监测。

不知道各位有什么好的方案可以推荐一下～

**Comment by masachi** [comment]
我是用 [v2ray-exporter](https://github.com/wi1dcard/v2ray-exporter) 然后搭配 Grafana 做的...  
接多个数据源 多个面板.. 
（但是原帖下面的Grafana面板好像不能用了， 然后重启服务时间长监控有概率断，因为没上MySQL 记录历史值）
